README.md

```diff
@@ -6,9 +6,37 @@ Another Validator of OpenAPI spec repository Configuration And Directories.
 
 NPM: https://www.npmjs.com/package/@azure/avocado
 
+## Overview
+
+Avocado validate folder structure and configuration for repository "Azure/azure-rest-api-specs". Avocado report new errors involved in PR. Avocado is a CI tool that will trigger automatically by azure DevOps pipeline when a new pull request is created.

```

Comment:
Can avocado tool used in other places given that it's open source? No need to scope the repository to `azure/azure-rest-api-specs`, it could be called out as an example. Try to describe the overall functionality this tool supports.

Reply:
Updated